core/services/fluxmonitor/flux_monitor.go

```diff
@@ -198,14 +200,27 @@ func (fm *concreteFluxMonitor) AddJob(job models.JobSpec) error {
 
 // RemoveJob stops and removes the checker for all Flux Monitor initiators belonging
 // to the passed job ID.
-func (fm *concreteFluxMonitor) RemoveJob(id *models.ID) {
+func (fm *ConcreteFluxMonitor) RemoveJob(id *models.ID) {
 	if id == nil {
 		logger.Warn("nil job ID passed to FluxMonitor#RemoveJob")
 		return
 	}
 	fm.chRemove <- *id
 }
 
+// XXXTestingOnlyCreateJob creates a job with a specific answer and round, for
+// testing nodes with malicious behavior
+func (fm *ConcreteFluxMonitor) XXXTestingOnlyCreateJob(t *testing.T,
```

Comment:
There's actually a better way to do this.  If you create a file called, say, `helpers_test.go`, and then at the top of the file, declare it to be a member of `package fluxmonitor`, you can put this declaration in there, and it will be available to your tests.  When the application is built normally, that file will not be included, and so the method won't be available.  We should already have a `helpers_test.go` in the FM package, come to think of it.  Check that out for some examples.

Reply:
I believe it will  only compile `*_test.go` files in the package under test. I tried it here, and got a compiler error. `helpers_test.go` symbols are only referenced in the `fluxmonitor` package.